installation/developer-installation/full-developer-installation.md

```diff
@@ -0,0 +1,247 @@
+---
+description: Setting up the recommended OctoBot development environment
+---
+
+# full developer installation
+
+{% hint style="info" %}
+This page is referring to OctoBot in versions superior to 0.4.0.
+{% endhint %}
+
+* ****[**Install OctoBot requirements**](https://app.gitbook.com/s/XGhS0OJcF37WFUAtCJrB/c/K2S34tUZ8NdXf7pYP47C/guides/dev-environment-setup#install-octobot-requirements)****
+* ****[**Cloning OctoBot repositories**](https://app.gitbook.com/s/XGhS0OJcF37WFUAtCJrB/c/K2S34tUZ8NdXf7pYP47C/guides/dev-environment-setup#cloning-octobot-repositories)****
+* ****[**OctoBot repositories**](https://app.gitbook.com/s/XGhS0OJcF37WFUAtCJrB/c/K2S34tUZ8NdXf7pYP47C/guides/dev-environment-setup#octobot-repositories)****
+* ****[**Setting up the IDE**](https://app.gitbook.com/s/XGhS0OJcF37WFUAtCJrB/c/K2S34tUZ8NdXf7pYP47C/guides/dev-environment-setup#setting-up-the-ide)****
+* ****[**create starting scripts in pycharm**](https://app.gitbook.com/s/XGhS0OJcF37WFUAtCJrB/c/K2S34tUZ8NdXf7pYP47C/installation/developer-installation/full-developer-installation#create-starting-scripts-in-pycharm)****
+* ****[**start OctoBot in pycharm**](https://app.gitbook.com/s/XGhS0OJcF37WFUAtCJrB/c/K2S34tUZ8NdXf7pYP47C/installation/developer-installation/full-developer-installation#start-octobot-in-pycharm)****
+
+## Install OctoBot requirements
+
+**Download and install:**
+
+* IDE: [PyCharm](https://www.jetbrains.com/pycharm/)
+* SCM: [Git](https://git-scm.com/downloads)
+*   we also use [GitKraken](https://www.gitkraken.com/git-client) to easily manage
+
+    OctoBot's multiple repos, this is just a quality of life
+
+    improvement and is not necessary.
+* Language: [Python 3.8](https://www.python.org/downloads/)
+
+#### Dependencies for Windows
+
+* download [Visual Studio build tools](https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2019) and install "Desktop development with C++"
+
+![](<../../.gitbook/assets/image (6).png>)
+
+![](<../../.gitbook/assets/image (1).png>)
```

Comment:
Can you rename your image assets to have a name that matches the image content please?